src/content/deployment/subctl.en.md

```diff
@@ -108,23 +108,32 @@ detected network plugin, and the detected Cluster and Service CIDRs.
 | `--kubeconfig` `<string>`    | Absolute path(s) to the kubeconfig file(s) (default "$HOME/.kube/config")
 | `--kubecontext` `<string>`   | Kubeconfig context to use
 
-### verify-connectivity
+### verify
 
-`subctl verify-connectivity <kubeConfig1> <kubeConfig2> [flags]`
+`subctl verify <kubeConfig1> <kubeConfig2> [flags]`
 
-The `verify-connectivity` command verifies dataplane connectivity between two clusters. The
+The `verify` command verifies a Submariner deployment between two clusters is functioning properly. The
 `kubeConfig1` file will be `ClusterA` in the reports, while `kubeConfig2` will be `ClusterB` in the
 reports. The `--verbose` flag is recommended to see what's happening during the tests.
 
-Dataplane connectivity is verified in multiple ways:
+There are several suites of verifications that can be performed. By default all verifications are performed.
```

Comment:
```suggestion
There are several suites of verifications that can be performed. By default all verifications are performed.
Some verifications are deemed disruptive in that they change some state of the clusters as a side effect.
If running the command interactively, you will be prompted for confirmation to perform disruptive 
verifications unless the `--enable-disruptive` flag is also specified. If running non-interactively (that is with no stdin), `--enable-disruptive` must be specified otherwise disruptive verifications are skipped.
```